tests/test_model_loading.py

```diff
@@ -0,0 +1,14 @@
+from jerboa.utils.load_model import load_model
+import pytest
+
+
+@pytest.mark.slow
```

Comment:
you might need to add a marker to the pyproject.toml https://github.com/docarray/docarray/blob/main/pyproject.toml